src/pretix/control/views/vouchers.py

```diff
@@ -283,6 +283,12 @@ def get_form_class(self):
                 form_class = response
         return form_class
 
+    def get_form(self, form_class=None):
+        form = super().get_form(form_class)
+        for f in form.fields.values():
+            f.disabled = True
```

Comment:
Doesn’t this disable all fields even if I have the permission `can_change_vouchers`? I did a quick test and cannot edit any voucher.

Reply:
m) yes, oversight on my end